packages/gusb.rb

```diff
@@ -3,16 +3,13 @@
 class Gusb < Package
   description 'GUsb is a GObject wrapper for libusb1'
   homepage 'https://www.openhub.net/p/gusb'
-  version '0.2.11'
+  version '0.2.11-0'
   source_url 'https://github.com/hughsie/libgusb/archive/gusb_0_2_11.tar.gz'
```

Comment:
Update available?  https://github.com/hughsie/libgusb/archive/0.3.0.tar.gz

Reply:
@uberhacker  No. I won't update it. I have tried that and get the errors, which I can not fix right now.

Reply:
No worries.  It looks more like a beta release anyway.